packages/coinstac-ui/app/render/state/ducks/maps.js

```diff
@@ -30,15 +34,33 @@ const INITIAL_STATE = {
 //   {type: 'boolean', name: 'isControl'},
 //   {type: 'number', name: 'age'}
 // ]
+//
+const getAllFiles = ((dirPath, arrayOfFiles) => {
+  const files = fs.readdirSync(dirPath)
+
+  arrayOfFiles = arrayOfFiles || [];
+
+  files.forEach((file) => {
+    if (fs.statSync(`${dirPath}/${file}`).isDirectory()) {
+      arrayOfFiles.push(path.join(__dirname, dirPath, '/', file));
+      arrayOfFiles = getAllFiles(`${dirPath}/${file}`, arrayOfFiles);
+    } else if (basename(file) !== '.DS_Store') {
+      arrayOfFiles.push(path.join(__dirname, dirPath, '/', file));
```

Comment:
to keep this section os agnostic avoid string concat'ing paths and don't use separators directly use `path.sep`, however the slashes in these joins aren't necessary, join does the work

Reply:
Fixed.